crates/matrix-sdk-crypto/src/types/requests/room_message.rs

```diff
@@ -28,5 +28,5 @@ pub struct RoomMessageRequest {
     pub txn_id: OwnedTransactionId,
 
     /// The event content to send.
-    pub content: AnyMessageLikeEventContent,
+    pub content: Box<AnyMessageLikeEventContent>,
```

Comment:
@Hywan it would be nice to document breaking changes like this in the changelog, where possible!

Reply:
Bleh, it was actually the change to `AnyOutgoingRequest::RoomMessage` which broke my build. Still, the point stands.

Reply:
Sure, will do next time.